src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNONative.m

```diff
@@ -115,9 +115,20 @@ bool uno_native_is_attached(NSView<UNONativeElement>* element)
 
 void uno_native_measure(NSView<UNONativeElement>* element, double childWidth, double childHeight, double availableWidth, double availableHeight, double* width, double* height)
 {
-    CGSize size = element.subviews.firstObject.frame.size;
```

Comment:
You should keep this line to get the size. It's both:

* faster, since you don't call the same selectors twice later; and
* ensure you get the a single value (in case something else, in another thread, change the size between the calls).